pkg/exchange/okex/okexapi/get_candles_request.go

```diff
@@ -0,0 +1,136 @@
+package okexapi
+
+//go:generate -command GetRequest requestgen -method GET -responseType .APIResponse -responseDataField Data
+//go:generate -command PostRequest requestgen -method POST -responseType .APIResponse -responseDataField Data
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"time"
+
+	"github.com/c9s/requestgen"
+
+	"github.com/c9s/bbgo/pkg/fixedpoint"
+	"github.com/c9s/bbgo/pkg/types"
+)
+
+type KLine struct {
+	StartTime    types.MillisecondTimestamp
+	OpenPrice    fixedpoint.Value
+	HighestPrice fixedpoint.Value
+	LowestPrice  fixedpoint.Value
+	ClosePrice   fixedpoint.Value
+	// Volume trading volume, with a unit of contract.cccccbcvefkeibbhtrebbfklrbetukhrgjgkiilufbde
+
+	// If it is a derivatives contract, the value is the number of contracts.
+	// If it is SPOT/MARGIN, the value is the quantity in base currency.
+	Volume fixedpoint.Value
+	// VolumeInCurrency trading volume, with a unit of currency.
+	// If it is a derivatives contract, the value is the number of base currency.
+	// If it is SPOT/MARGIN, the value is the quantity in quote currency.
+	VolumeInCurrency fixedpoint.Value
+	// VolumeInCurrencyQuote Trading volume, the value is the quantity in quote currency
+	// e.g. The unit is USDT for BTC-USDT and BTC-USDT-SWAP;
+	// The unit is USD for BTC-USD-SWAP
+	VolumeInCurrencyQuote fixedpoint.Value
```

Comment:
this field could overflow, maybe use float64 for now?

Reply:
or maybe not now :p

Reply:
OK, i will skip it